examples/gnrc_border_router/README.md

```diff
@@ -60,3 +60,98 @@ for further help.
+In order to select ethos as the driver, you should modify some lines as following:
+```make
+#CFLAGS += -DSLIP_UART=$(SLIP_UART)
+#CFLAGS += -DSLIP_BAUDRATE=$(SLIP_BAUDRATE)
+CFLAGS += '-DETHOS_UART=UART_DEV(0)' -DETHOS_BAUDRATE=115200 -DUSE_ETHOS_FOR_STDIO
+```
+Then deactivate the slip module and activate ethos
+```make
+# Include SLIP package for IP over Serial communication
+#USEMODULE += gnrc_slip
+USEMODULE += ethos gnrc_netdev2
+```
+This will allow to compile the BR firmware using ethos.
+Thus:
+```bash
+$ make clean all flash
+```
+should compile the example and flash it to your board.
+
+Afterwards, you need to compile the ethos tool instead of tunslip.
+To do this, in `dist/tools/ethos` you'll find the tool, just compile it with:
+```bash
+$ make
+```
+
+## Usage
+Once you flashed the BR example on your board, you should intialize your IP settings by choosing a prefix for your 6lo network. Just type `make term` and setup your BR:
+```
+> ifconfig 6 add affe::2/48
+> ifconfig 5 add affe::3/64
+> fibroute add :: via <link-local of tap> dev 6
+```
+Here we have choosen `affe` as the prefix. It should be disseminated to your other nodes on the radio range.
+
+Next, to have access to the nodes attached to the BR from your linux machine, it is necessary to add a `tap` interface as well as configure it to route the packets coming from your 6lo network.
+As an example, you can do the following:
+```bash
+$ sudo ip tuntap add tap0 mode tap user ${USER}
+```
+This will setup your `tap` interface, then:
+```bash
+$ sudo ip link set tap0 up
+$ sudo ip a a affe::1/48 dev tap0
+$ sudo ip r d affe::/48 dev tap0
+$ sudo ip r a affe::2 dev tap0
+$ sudo ip r a affe::/48 via affe::2 dev tap0
+```
+will configure the interface and add some routes in order to have access to the 6lo network from Linux.
+
+Finally, run ethos by choosing the right serial interface (`/dev/ttyUSB*` or `/dev/ttyACM*` according to your board):
+```bash
+$ sudo ./ethos tap0 /dev/ttyACM0
+```
+After doing this, you should be able to ping your nodes using the global addres starting by your prefix, on our case `affe::`.
+
+# gnrc_border_router using automatic configuration (not merged yet)
+In \#4725 (https://github.com/RIOT-OS/RIOT/pull/4725) an effort to automate all the previous steps is done using 3 tools:
+- ethos
+- uhcp
+- start_network.sh
+
+This will enable a *ready-to-use* environment in only one command.
+
+## Setup
+Merge \#4725 to your current branch.
+Afterwards, compile `ethos` as explained previously.
+Then, you need to compile UHCP (micro host control protocol). This tool is found in `/dist/tools/uhcp`. So, as usual:
+```
+$ make clean all
+```
+
+At this point you should have these two programs compiled, necessary to compile your gnrc_border_router ** DO NOT COMPILE IT BEFORE ethos AND uhcp **.
+Then, proceed to compile and flash gnrc_border_router to your board as explained before.
```

Comment:
In the web editor looked well, I don't know why in the diff it is formatter that way...